src/notebook_index.ipynb

```diff
@@ -163,7 +168,7 @@
     "<a id='examples.surrmod.alamo'></a>\n",
     "\n",
     "#### ALAMO\n",
-    "Automated Learning of Algebraic Models, see https://www.minlp.com/alamo-modeling-tool\n",
+    "Automated Learning of Algebraic Models, see https://www.minlp.com/alamo\n",
```

Comment:
I believe this is one of the links fixed by @blnicho in #76, so the URL ending in `alamo-modeling-tool` is the correct one.

```suggestion
    "Automated Learning of Algebraic Models, see https://www.minlp.com/alamo-modeling-tool\n",
```

Reply:
I will fix this link and the ones below.